bin.src/createDetGeom.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
+#createRefSpec.py '/Users/azuri/spectra/pfs/PFS' --id visit=4 filter='r' spectrograph=2 site='F' category='A' -C ../obs_pfs/config/pfs/createRefSpec.py --output "/Users/azuri/spectra/pfs/PFS" --clobber-config
+from pfs.drp.stella.createDetGeom import CreateDetGeomTask
+task = CreateDetGeomTask()
+task.run()
```

Comment:
I see from the discussion on HipChat what happened here: this is a "command line task" without being a `CmdLineTask`, and hence the weird set up. I agree with the commentary from @RobertLuptonTheGood on HipChat: this probably more convoluted than it needs to be, but it's not worth spending effort in refactoring so long as this does the job.

I do think it would be useful to add a comment describing what's going on, though, either to this file or to `createDetGeom.py` itself.